content/blog/unit-testing-assets/index.md

```diff
@@ -0,0 +1,195 @@
+---
+title: "Unit Testing Assets"
+date: 2020-06-10T11:08:26-07:00
+draft: false
+meta_desc: "Learn how unit testing your infrastructure assets can ensure help correctness of your deployments."
+meta_image: meta.png
+authors:
+    - lee-zen
+tags:
+    - Testing
+    - Infrastructure
+---
+
+When deploying infrastructure, we want to ensure that what we're deploying matches our expectations.
+One way to do so is via [unit testing]({{< relref "/docs/guides/testing/unit" >}}).
+We've talked about this concept in previous posts such as [in this overview]({{< relref "unit-test-infrastructure" >}})
```

Comment:
Best to fully qualify. It's unlikely we'll add `unit-test-infrastructure` elsewhere, but if we do, Hugo will get confused.

```suggestion
We've talked about this concept in previous posts such as [in this overview]({{< relref "/blog/unit-test-infrastructure" >}})
```